tests/qasm/test_qasm_to_mpqp.py

```diff
@@ -49,8 +49,8 @@
                 H(0),
                 H(1),
                 CNOT(0, 1),
-                BasisMeasure([0]),
-                BasisMeasure([1]),
+                BasisMeasure([0], [0]),
```

Comment:
wonrg

Reply:
No, because we are checking the BasisMeasure repr but as it is not associated with a circuit, is not dynamic. So we need to specify the c_targets.